scripts/package

```diff
@@ -38,4 +49,16 @@ if [ ! -d build/system-charts ]; then
     mkdir -p build
     git clone --branch $SYSTEM_CHART_DEFAULT_BRANCH https://github.com/rancher/system-charts build/system-charts
 fi
-TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go build/system-charts $IMAGE $AGENT_IMAGE
+
+if [ ! -d build/charts ]; then
+    mkdir -p build
+    git clone --branch $CHART_DEFAULT_BRANCH https://github.com/rancher/charts build/charts
+
+    # Iterate through chart directories and execute callback to extract latest version tgz
+    find build/charts/assets -type d -maxdepth 1 -exec bash -c 'extract_latest_version_tgz {}' \;
+
+    # Remove index to force building a virtual index like system charts
+    rm -f build/charts/index.yaml build/charts/assets/index.yaml
```

Comment:
Didn't realize the walker script actually generated a virtual index

Reply:
We build a virtual index (in helm package) when one isn't found. When building it we set `version.Dir` for each version to it's location, which is later used to pick the images. Since charts is a helm repo, `version.Dir` is empty and `version.URLs` contains the location of the tgz instead. This was the cleanest way to reuse all the logic for system-charts. A little hacky so left a comment for the future.